judge/admin/problem.py

```diff
@@ -245,3 +245,16 @@ def construct_change_message(self, request, form, *args, **kwargs):
         if form.cleaned_data.get('change_message'):
             return form.cleaned_data['change_message']
         return super(ProblemAdmin, self).construct_change_message(request, form, *args, **kwargs)
+
+
+class ProblemPointsVoteAdmin(admin.ModelAdmin):
+    list_display = ('points', 'voter', 'problem', 'note')
```

Comment:
Don't show the `note` in the list view because it can be arbitrarily long.

Reply:
Removed.